functional/ek-cert-use-ek_check_script/test.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+HTTP_SERVER_PORT=8080
+AGENT_ID="d432fbb3-d2f1-4a97-9ef7-75bd81c00000"
+
+rlJournalStart
+
+    rlPhaseStartSetup "Do the keylime setup"
+        rlRun 'rlImport "./test-helpers"' || rlDie "cannot import keylime-tests/test-helpers library"
+        rlAssertRpm keylime
+        # update /etc/keylime.conf
+        limeBackupConfig
+        # tenant, set to true to verify ek on TPM
+        rlRun "limeUpdateConf tenant require_ek_cert false"
+        # if TPM emulator is present
+        if limeTPMEmulated; then
+            # start tpm emulator
+            rlRun "limeStartTPMEmulator"
+            rlRun "limeWaitForTPMEmulator"
+            # make sure tpm2-abrmd is running
+            rlServiceStart tpm2-abrmd
+            sleep 5
+            # start ima emulator
+            rlRun "limeInstallIMAConfig"
+            rlRun "limeStartIMAEmulator"
+        fi
+        
+        # start keylime_verifier
+        rlRun "limeStartVerifier"
+        rlRun "limeWaitForVerifier"
+        rlRun "limeStartRegistrar"
+        rlRun "limeWaitForRegistrar"
+        rlRun "limeStartAgent"
+        rlRun "limeWaitForAgentRegistration ${AGENT_ID}"
+        # create allowlist and excludelist
+        limeCreateTestLists
+    rlPhaseEnd
+
+    # its possible, thats need to be need in different test scenario
+    rlPhaseStartTest "Add keylime agent and check genuine of TPM via ek_check_script option"
+        rlRun "cat > /var/lib/keylime/check_ek_script.sh <<_EOF
+#!/bin/sh
+echo AGENT_UUID=$AGENT_UUID
+echo ENDORSEMENT_KEY=$EK
+echo ENDORSEMENT_KEY_TPM=$EK_TPM
+echo ENDORSEMENT_KEY_CERTIFICATES=$EK_CERT
+echo PROVKEYS=$PROVKEYS
+_EOF" 
+        rlRun "chmod 755 /var/lib/keylime/check_ek_script.sh"
```

Comment:
I think better to set this script to be owned by `keylime.keylime` and have permissions `500`.